src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringParts = sourceString.split(';');
+  const strWithoutSpaces = sourceStringParts.map((str) => str.trim());
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes the code much more readable. Consider chaining `map` directly after the `split` method with proper indentation.